libpod/runtime_img.go

```diff
@@ -325,6 +325,11 @@ func (r *Runtime) LoadImageFromSingleImageArchive(ctx context.Context, writer io
 	return "", errors.Wrapf(saveErr, "error pulling image")
 }
 
+func (r *Runtime) RemoveImageFromStorage(id string) error {
```

Comment:
This needs a comment on how this is potentially dangerous and should be avoided in favor of normal image removal whenever possible.